src/components/Training/Modal.js

```diff
@@ -153,8 +153,12 @@ const Modal = ({ content, toggleModal }) => (
         </Close>
         <Grid>
           <Row>
-            <CourseInfo content={content} />
-            <CourseContent content={content} />
+            <Col width={[1, 1, 1, 1, 1 / 2]}>
+              <CourseInfo content={content} />
```

Comment:
with these lines 156-161, won't we have columns inside columns? is this what we want?

Reply:
I'm all ears, if you have other suggestions?

I see what you mean, with it seeming strange at first glacen. But the `Col` component is just a div underneath, so I guess there's no real problem there. And I figure this captures the intent nicely enough.

You can either think of it like:
1. split the page in half (6 columns and 6 columns)
2. allow content inside that half of the page span 5 columns

I reckon that version is more like how a designer thinks?

Or you can think of it like:
- text spans 5 columns
- 1 column of space
- text spans 5 columns
- 1 column of space

If we opt for this, then I guess we would have to create `div`s with nothing in them, to represent the column of space.